x/paloma/keeper/keeper.go

```diff
@@ -103,8 +110,12 @@ func (k Keeper) JailValidatorsWithMissingExternalChainInfos(ctx sdk.Context) err
 
 		sort.Strings(notSupported)
 
+		if err != nil {
+			g.Add(err)
+			continue
+		}
```

Comment:
This is unneeded, no?

Reply:
should we ignore this err from `utilkeeper.ValAddressFromBech32` ? or any other handler needed?

Reply:
You define `err` in line 92: `exts, err := k.Valset.GetValidatorChainInfos(ctx, valAddr)`, and you perform the proper error checking on line 93.

As far as I can tell, `err` is not redefined before this code is reached, so this check is superfluous.

Reply:
so ignoring the `err` from valAddr is fine?  in line 87
````go
valAddr, _ := utilkeeper.ValAddressFromBech32(k.AddressCodec, val.GetOperator())
`````

Reply:
No. I'm not asking you to ignore any errors. Let me add some context to what I mean:

```go
for _, val := range vals {
                // err declared
		valAddr, err := utilkeeper.ValAddressFromBech32(k.AddressCodec, val.GetOperator())

                // err checked
		if err != nil {
			g.Add(err)
			continue
		}

                // err declared
		exts, err := k.Valset.GetValidatorChainInfos(ctx, valAddr)

                // err checked
		if err != nil {
			g.Add(err)
			continue
		}

                // err is not redeclared within the following code block
		valmap := make(map[mapkey]struct{})
		for _, ext := range exts {
			key := mapkey{ext.GetChainType(), ext.GetChainReferenceID()}
			valmap[key] = struct{}{}
		}

		notSupported := []string{}
		for mustExistKey := range mmap {
			if _, ok := valmap[mustExistKey]; !ok {
				// well well well
				notSupported = append(notSupported, fmt.Sprintf("[%s, %s]", mustExistKey[0], mustExistKey[1]))
			}
		}

		sort.Strings(notSupported)

                // superfluous error check. We already checked this value, and it hasn't been redeclared since.
		if err != nil {
			g.Add(err)
			continue
		}

		if len(notSupported) > 0 {
			g.Add(k.Valset.Jail(ctx, valAddr, fmt.Sprintf(types.JailReasonNotSupportingTheseExternalChains, strings.Join(notSupported, ", "))))
		}
	}
```

Also, probably also remove this `well well well` comment ;)

Reply:
removed the superfluous error check and `well well well`